src/components/mobile/mobile.vue

```diff
@@ -3,26 +3,26 @@
     <template #header>
       <div
         class="
-          x-list x-list--horizontal x-list__item--expand x-list--padding-03 x-list--align-center
+          x-list x-list--horizontal x-list__item--expand x-list--padding-05 x-list--align-center
         "
       >
-        <Close class="x-button--ghost">
-          <ArrowLeftIcon />
+        <Close class="x-button--ghost x-padding--left-00">
+          <ArrowLeftIcon class="x-icon--l" />
         </Close>
         <SearchBox class="x-list__item--expand" />
       </div>
     </template>
 
     <template #sub-header>
-      <div v-if="$x.relatedTags.length > 0" class="x-list x-list--padding-03">
+      <div v-if="$x.relatedTags.length > 0" class="x-list x-list--padding-05 x-padding--top-00">
         <LocationProvider location="predictive_layer">
           <RelatedTags />
         </LocationProvider>
       </div>
     </template>
 
     <template #toolbar>
-      <MobileToolbar />
+      <MobileToolbar class="x-padding--05 x-padding--top-00" />
```

Comment:
Yep, that would be ideal, but I set it that way because some elements like the related tags in the grid or the partial results will occupy the whole width.

Reply:
Don't think of this as a Setup. This is an Archetype that must be configured for any kind of setup. And in any kind of setup may be required to have different padding/spacing in different elements.

Here, in the Archetype, it is not so bad to repeat yourself and avoid abstractions. Because you don't know what will be needed in the setup tomorrow.

So it is ok as it is.